src/StatusQ/Controls/StatusSwitchTab.qml

```diff
@@ -0,0 +1,53 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+import QtGraphicalEffects 1.13
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+TabButton {
+    id: control
+    contentItem: Item {
+        height: 36
+        MouseArea {
+            id: sensor
+            hoverEnabled: true
+            anchors.fill: parent
+            cursorShape: control.hovered ? Qt.PointingHandCursor : Qt.ArrowCursor
```

Comment:
if its `pressed` its arrow?

Reply:
Not sure I understand the question. It's not right now.